admin_guide/image_signatures.adoc

```diff
@@ -53,27 +54,113 @@ The `atomic` command line interface (CLI), version 1.12.5 or greater, provides
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/signing_container_images[RHEL Atomic Host Managing Containers: Signing Container Images] documentation
+or the `atomic push --help` output for argument details.
 
 A specific example workflow of working with the `atomic` CLI and an OpenShift
 Container Registry is documented in the
 link:https://access.redhat.com/articles/2750891#working-with-openshift-and-atomic-registry[Container Image Signing Integration Guide].
 
+[[verifying-image-signatures-using-openshift-cli]]
+== Verifying Image Signatures Using OpenShift CLI
+
+You can verify the signatures of an image imported to an OpenShift Container
+Registry using the `oc adm verify-image-signature` command. This command
+verifies if the image identity contained in the image signature can be trusted
+by using the public GPG key to verify the signature itself then match the
+provided expected identity with the identity (the pull spec) of the given image.
+
+By default, this command uses the public GPG keyring located in
+*_$GNUPGHOME/pubring.gpg_*, typically in path *_~/.gnupg_*. By default, this
+command does not save the result of the verification back to the image object.
+To do so, you must specify the `--save` flag, as shown below.
+
+[NOTE]
+====
+In order to verify the signature of an image, the user must have the
+`image-auditor` cluster role.
+ifdef::openshift-origin,openshift-enterprise[]
+Cluster administrators can add this using:
+
+----
+$ oc adm policy add-cluster-role-to-user system:image-auditor <user_name>
+----
+endif::[]
+====
+
+Using the `--save` flag on already verified image together with invalid GPG key
+or invalid expected identity causes the saved verification status to be removed,
+and the image will become unverified.
+
+To verify an image signature use the following format:
+
+----
+$ oc adm verify-image-signature <image> --expected-identity=<pull_spec> [--save] [options]
+----
+
+The `<pull_spec`> can be found by describing the image stream.
+The `<image>` may be found by describing the image stream tag.
+See the following example command output.
+
+.Example Image Signature Verification
+----
+$ oc describe is nodejs -n openshift
+Name:             nodejs
+Namespace:        openshift
+Created:          2 weeks ago
+Labels:           <none>
+Annotations:      openshift.io/display-name=Node.js
+                  openshift.io/image.dockerRepositoryCheck=2017-07-05T18:24:01Z
+Docker Pull Spec: 172.30.1.1:5000/openshift/nodejs
+...
+
+$ oc describe istag nodejs:latest -n openshift
+Image Name:	sha256:2bba968aedb7dd2aafe5fa8c7453f5ac36a0b9639f1bf5b03f95de325238b288
+...
+
+$ oc adm verify-image-signature \
+    sha256:2bba968aedb7dd2aafe5fa8c7453f5ac36a0b9639f1bf5b03f95de325238b288 \
+    --expected-identity 172.30.1.1:5000/openshift/nodejs:latest \
+    --public-key /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release \
+    --save
+----
+
```

Comment:
All the imagestream in the openshift project don't have any signatures, when follow the steps will get error: error: xxx does not have any signature